observability/logging/logging-6.0/log6x-about.adoc

```diff
@@ -165,3 +181,154 @@ spec:
 ----
 
 . Verify that logs are visible in the Log section of the Observe tab in the OpenShift web console.
+
+=== Quick Start with OTEL
+
+To configure OTLP ingestion and enable the OTEL data model, follow these steps:
+
+:FeatureName: The OpenTelemetry Protocol (OTLP) output log forwarder
+include::snippets/technology-preview.adoc[]
+
+.Prerequisites
+* Cluster administrator permissions
+
+.Procedure
+
+. Install the `OpenShift Logging` and `Loki` Operators from OperatorHub.
+
+. Create a `LokiStack` custom resource (CR) in the `openshift-logging` namespace:
++
+[source,yaml]
+----
+apiVersion: loki.grafana.com/v1
+kind: LokiStack
+metadata:
+  name: logging-loki
+  namespace: openshift-logging
+spec:
+  managementState: Managed
+  size: 1x.extra-small
+  storage:
+    schemas:
+    - effectiveDate: '2022-06-01'
+      version: v13
+    secret:
+      name: logging-loki-s3
+      type: s3
+    storageClassName: gp3-csi
+  tenants:
+    mode: openshift-logging
+----
+
+. Create a service account for the collector:
++
+[source,shell]
+----
+$ oc create sa collector -n openshift-logging
+----
+
+. Create a `ClusterRole` for the collector:
++
+[source,yaml]
+----
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: logging-collector-logs-writer
+rules:
+- apiGroups:
+  - loki.grafana.com
+  resourceNames:
+  - logs
+  resources:
+  - application
+  - audit
+  - infrastructure
+  verbs:
+  - create
+----
+
+. Bind the `ClusterRole` to the service account:
++
+[source,shell]
+----
+$ oc adm policy add-cluster-role-to-user logging-collector-logs-writer -z collector
+----
+
+. Install the Cluster Observability Operator.
+
+. Create a `UIPlugin` to enable the Log section in the Observe tab:
++
+[source,yaml]
+----
+apiVersion: observability.openshift.io/v1alpha1
+kind: UIPlugin
+metadata:
+  name: logging
+spec:
+  type: Logging
+  logging:
+    lokiStack:
+      name: logging-loki
+----
+
+. Add additional roles to the collector service account:
++
+[source,shell]
+----
+$ oc project openshift-logging
+$ oc adm policy add-cluster-role-to-user collect-application-logs -z collector
+$ oc adm policy add-cluster-role-to-user collect-audit-logs -z collector
+$ oc adm policy add-cluster-role-to-user collect-infrastructure-logs -z collector
+----
+
+. Create a `ClusterLogForwarder` CR to configure log forwarding:
++
+[source,yaml]
+----
+apiVersion: observability.openshift.io/v1
+kind: ClusterLogForwarder
+metadata:
+  name: my-logforwarder
+  namespace: my-app-namespace
```

Comment:
We should put this into the `openshift-logging` namespace. Maybe it also makes sense to converge on a "default name" in the documentation. Previously we used `instance`, so we probably want to avoid that one. Maybe "clusterforwarder" or just "collector" are options?

```suggestion
  name: my-logforwarder
  namespace: openshift-logging
```